eng/Build.ps1

```diff
@@ -605,6 +612,39 @@ try {
         throw "Error Verifying nupkgs have access to the source code"
     }
 
+    $verifypackageshipstatusFailed = $false
+    if ($verifypackageshipstatus) {
```

Comment:
Related to Vlad's comment, pull this out into a separate file, something like `eng/VerifyPackageShipStatus.ps1`, that way it'll be easier to add as a separate build step and/or run locally.

Reply:
There is a build.ps1 switch for running this:
https://github.com/dotnet/fsharp/pull/13793/files#diff-9841a029538b042b58ee9aabac3d4cd9c725a2495d81d188d23f6ad617f2a262R66

https://github.com/dotnet/fsharp/pull/13793/files#diff-9841a029538b042b58ee9aabac3d4cd9c725a2495d81d188d23f6ad617f2a262R121

So build -verifypackageshipstatus   will make it easy to run.